### ISSUE TITLE
false negative for direct returns inside wrapper functions

### ISSUE THREAD

**Issue by cwrau** [tomarrell/wrapcheck#55]
The following functions;

```go
func (s *Service) ListLoadBalancers3(ctx context.Context) ([]loadbalancers.LoadBalancer, error) {
	return util.WithSpan(ctx, s.loadBalancerClient.TracerName, "ListLoadBalancers",
		func(ctx context.Context, span trace.Span) ([]loadbalancers.LoadBalancer, error) {
			return cloudUtil.GetItems(ctx, s.loadBalancerClient.ServiceClient,
				loadbalancers.ListOptsBuilder(
					loadbalancers.ListOpts{},
				),
				loadbalancers.List,
				loadbalancers.ExtractLoadBalancers,
			)
		},
	)
}

func (s *Service) ListLoadBalancers4(ctx context.Context) ([]loadbalancers.LoadBalancer, error) {
	return cloudUtil.GetItems(ctx, s.loadBalancerClient.ServiceClient,
		loadbalancers.ListOptsBuilder(
			loadbalancers.ListOpts{},
		),
		loadbalancers.List,
		loadbalancers.ExtractLoadBalancers,
	)
}
```

have different linting errors, the upper one only reports on `util.withSpan` and not on `cloudUtil.GetItems` while the lower one correctly identifies `cloudUtil.GetItems`.

If I ignore `util.withSpan` the error vanishes, but there is still no report on `cloudUtil.getItems`.

If I do the following;

```go
func (s *Service) ListLoadBalancers2(ctx context.Context) ([]loadbalancers.LoadBalancer, error) {
	return util.WithSpan(ctx, s.loadBalancerClient.TracerName, "ListLoadBalancers",
		func(ctx context.Context, span trace.Span) ([]loadbalancers.LoadBalancer, error) {
			loadBalancers, err := cloudUtil.GetItems(ctx, s.loadBalancerClient.ServiceClient,
				loadbalancers.ListOptsBuilder(
					loadbalancers.ListOpts{},
				),
				loadbalancers.List,
				loadbalancers.ExtractLoadBalancers,
			)
			if err != nil {
				return nil, err
			}
			return loadBalancers, nil
		},
	)
}
```

the linter correctly identifies the `return nil, err` via `cloudUtil.GetItems`.

So seemingly "direct returns" or whatever this is called are not being detected if it is inside an (ignored) wrapper function.

As nearly all my functions use wrapper functions, for tracing, this is quite the problem.

Is there a setting I need to set or is this a bug?
